test/unit/tracks/text-track-controls.test.js

```diff
@@ -364,6 +364,8 @@ QUnit.test('menu items should polyfill mode change events', function(assert) {
   assert.equal(changes, 2, 'clicks trigger change events');
 
   player.dispose();
+  trackMenuItem.dispose();
+  player.textTracks().off('change');
```

Comment:
I almost wonder if we should call `off` on track lists when we dispose, instead of doing this.

Reply:
probably, though, I thought we did that?

Reply:
I think we clear them out of `Tracks`. Let's see...

Reply:
Doesn't seem like it: https://github.com/videojs/video.js/blob/master/src/js/tech/tech.js#L367

Reply:
Maybe we want to call `off` on them on player dispose, rather than tech dispose, since the track lists stick around.

Reply:
I will remove the test changes related to this if #5867